packages/frontend/@n8n/chat/src/components/Input.vue

```diff
@@ -381,53 +388,55 @@ function adjustTextAreaHeight() {
 	}
 }
 .chat-inputs {
-	width: 100%;
+	width: var(--chat--input--width, 100%);
 	display: flex;
 	justify-content: center;
 	align-items: flex-end;
+	background: var(--chat--input--container--background, var(--color--background--light-2));
+	border: var(--chat--input--container--border, 1px solid var(--color--foreground--tint-1));
+	border-radius: var(--chat--input--container--border-radius, 24px);
+	padding: var(--chat--input--container--padding, 12px);
 
 	textarea {
 		font-family: inherit;
 		font-size: var(--chat--input--font-size);
 		width: 100%;
-		border: var(--chat--input--border, 0);
-		border-radius: var(--chat--input--border-radius);
-		padding: var(--chat--input--padding);
-		min-height: var(--chat--textarea--height, 2.5rem); // Set a smaller initial height
+		border: none;
+		border-radius: var(--chat--input--border-radius, 16px);
+		padding: var(--chat--input--padding, 12px 16px);
+		min-height: var(--chat--textarea--height, 2.5rem);
 		max-height: var(--chat--textarea--max-height);
-		height: var(--chat--textarea--height, 2.5rem); // Set initial height same as min-height
+		height: var(--chat--textarea--height, 2.5rem);
 		resize: none;
 		overflow-y: auto;
-		background: var(--chat--input--background, white);
+		background: transparent;
 		color: var(--chat--input--text-color, initial);
 		outline: none;
 		line-height: var(--chat--input--line-height, 1.5);
 
 		&::placeholder {
 			font-size: var(--chat--input--placeholder--font-size, var(--chat--input--font-size));
 		}
-		&:focus,
-		&:hover {
-			border-color: var(--chat--input--border-active, 0);
-		}
 	}
 }
 .chat-inputs-controls {
 	display: flex;
 }
 .chat-input-send-button,
 .chat-input-file-button {
-	height: var(--chat--textarea--height);
-	width: var(--chat--textarea--height);
-	background: var(--chat--input--send--button--background, white);
+	height: calc(var(--chat--textarea--height, 2.5rem) - 16px);
```

Comment:
is this backwards compatible? Why subtract 16px from it? Maybe a comment here could help.

Keep in mind this will go live affecting all users using the hosted chat with every n8n version that supports this.